third_party/rust/chromium_crates_io/Cargo.lock

```diff
@@ -873,7 +873,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 dependencies = [
  "proc-macro2",
  "quote",
- "syn 2.0.96",
+ "syn 2.0.98",
```

Comment:
no update to Cargo.toml?

Reply:
Cargo.toml got updated here https://github.com/brave/brave-core/pull/27522/commits/612fb32f5833da886a4c135e81100bde76983d8c#diff-49def88bdf9c3acd4363008a0d2651030d6f070aacf42e835c1c30883ece32baL538 and Cargo.lock got updated with that version in a bunch of places, but for some reason the dependency for `futures-macro` didn't get updated.